pages/api/graphql.js

```diff
@@ -0,0 +1,30 @@
+import { ApolloServer, gql } from 'apollo-server-micro';
+import dbConnect from 'utils/dbConnect';
+
+import User from 'models/User';
+import Space from 'models/Spaces';
+import { typeDefs, resolvers } from 'utils/schema';
+const apolloServer = new ApolloServer({ typeDefs, resolvers });
+
+const startServer = apolloServer.start();
+
+export default async function handler(req, res) {
+  res.setHeader('Access-Control-Allow-Credentials', 'true');
+  res.setHeader('Access-Control-Allow-Origin', 'https://studio.apollographql.com');
+  res.setHeader('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
+  if (req.method === 'OPTIONS') {
+    res.end();
+    return false;
+  }
+
+  await startServer;
```

Comment:
Was this in your tutorial as well? Normally, you're supposed to execute a function with the await keyword. No function is being executed if you await `startServer` which is probably just an object/value of some kind. Correct me if I'm wrong. If you're confused as to what I mean, I can explain later. Otherwise, can you send me the link to the tutorial you were looking at?